packages/neos-ui-contentrepository/src/registry/NodeTypesRegistry.js

```diff
@@ -163,4 +163,17 @@ export default class NodeTypesRegistry extends SynchronousRegistry {
 
         return viewConfiguration;
     }
+
+    getInlineEditorForProperty(nodeTypeName, propertyName) {
+        const nodeType = this.get(nodeTypeName);
+
+        return $get(['properties', propertyName, 'ui', 'inline', 'editor'], nodeType) || 'ckeditor';
+    }
+
+    getInlineEditorOptionsForProperty(nodeTypeName, propertyName) {
+        const nodeType = this.get(nodeTypeName);
+
+        return $get(['properties', propertyName, 'ui', 'inline', 'editorOptions'], nodeType) ||
```

Comment:
this additional configuration option is quite "ad-hoc"; we need to document these in a central place. Maybe for now "README.md"?